### ISSUE TITLE
Unable to open configuration file ...

### ISSUE THREAD

**Issue by maximilianosinski** [joelverhagen/TorSharp#98]
Settings code:
```
        var port = new Random().Next(40000, 50000);
        var settings = new TorSharpSettings
        {
            PrivoxySettings = { Disable = true },
            ExtractedToolsDirectory = $"ext{port}",
            ZippedToolsDirectory = $"zip{port}",
            TorSettings =
            {
                SocksPort = port,
                ControlPort = port + 1
            }
        };
```

Log:
`Feb 02 23:20:05.773 [warn] Unable to open configuration file "C:\Users\maxim\RiderProjects\yota-dos\YotaDos\bin\Debug\net6.0\ext40332\tor-win64-12.0.2\tor\ext40332\tor-win64-12.0.2\data\tor\torrc".`

**seems like a bug**

**Comment by joelverhagen** [comment]
Yeah, that's a bug. Currently the code expects fully-qualified paths. This seems to work fine:

```csharp
var port = new Random().Next(40000, 50000);
var settings = new TorSharpSettings
{
    PrivoxySettings = { Disable = true },
    ExtractedToolsDirectory = Path.GetFullPath($"ext{port}"),
    ZippedToolsDirectory = Path.GetFullPath($"zip{port}"),
    TorSettings =
            {
                SocksPort = port,
                ControlPort = port + 1
            }
};
```

**Comment by joelverhagen** [comment]
This should be fixed now in 2.14.0.
